plugin-docker/che-plugin-docker-runner/src/main/java/org/eclipse/che/plugin/docker/runner/BaseDockerRunner.java

```diff
@@ -319,7 +319,18 @@ protected ApplicationProcess newApplicationProcess(DeploymentSources toDeploy, R
                         unpackedApplication = unpackArchive(application, workDir, applicationFilename + "_unpack");
                     }
                 }
-                hostConfig.setBinds(new String[]{String.format("%s:%s", unpackedApplication.getAbsolutePath(), applicationBindDir)});
+
+                // On Windows binding directory needs to follow URL convention with first / and no colon :
```

Comment:
is https://github.com/boot2docker/boot2docker/blob/master/README.md#virtualbox-guest-additions  ok ?
Also I was using the try and debug mode